ovh/resource_iam_resource_tags.go

```diff
@@ -0,0 +1,218 @@
+package ovh
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/ovh/terraform-provider-ovh/v2/ovh/helpers"
+)
+
+func resourceIamResourceTags() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIamResourceTagsCreate,
+		ReadContext:   resourceIamResourceTagsRead,
+		UpdateContext: resourceIamResourceTagsUpdate,
+		DeleteContext: resourceIamResourceTagsDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceIamResourceTagsImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"resource_urn": {
```

Comment:
Maybe this should be renamed `urn` instead of `resource_urn` which is somewhat redundant, isn't it?